src/Kriti/CustomFunctions.hs

```diff
@@ -144,6 +150,11 @@ concatF = parserToFunc $ J.withArray "Array" \as -> do
       o = J.Object . fold . reverse <$> traverse (J.withObject "Nested Object" pure) l
   a <|> s <|> o
 
+notF :: KritiFunc
+notF = parserToFunc $ J.withBool "Bool" \case
+  False -> pure $ J.Bool True
+  True -> pure $ J.Bool False
```

Comment:
Would truthy operations make sense like `not null == true`?

Reply:
We could do that but it should go in a seperate PR.